fahrschein/src/main/java/org/zalando/fahrschein/JsonParserHelper.java

```diff
@@ -0,0 +1,32 @@
+package org.zalando.fahrschein;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+
+import javax.annotation.Nullable;
+import java.io.EOFException;
+import java.io.IOException;
+import java.io.InterruptedIOException;
+
+class JsonParserHelper {
+    private JsonParserHelper() {
+    }
+
+    static void expectToken(JsonParser jsonParser, JsonToken expectedToken) throws IOException {
```

Comment:
does it make sense to mark this as `public`?

Reply:
Should be used only internally. Also could not think of a better name than `JsonParserHelper` but for a non-public class this is probably not that important.